### DISCUSSION TITLE
Zeek container missing from sensor

### DISCUSSION THREAD

**Discussion by lerou114** [Security-Onion-Solutions/securityonion#6535] [Unsupported Versions]
I am running a distributed grid using version 2.3.90. I have a sensor that is showing up with a so-status:

![image](https://user-images.githubusercontent.com/8277856/145842719-a04d1a1e-f5f2-43f1-9faf-09b8bbf4f1a6.png)

When I run salt-call state.highstate on the sensor it is showing 2 errors:
![image](https://user-images.githubusercontent.com/8277856/145842929-ca0d6114-7f9a-4ed6-8964-9c8122ff30eb.png)

I am wondering where to start looking to troubleshoot this issue?

**Comment by weslambert** [comment]
What are the two errors displayed in the Salt output?

**Reply by lerou114** [reply]
I apologize I thought I had posted these previously. 

![image](https://user-images.githubusercontent.com/8277856/146065698-87421d32-b09b-459c-bf47-56a9b3cd9e5a.png)
![image](https://user-images.githubusercontent.com/8277856/146065869-423b94ce-cc22-4661-a641-af25123b9465.png)

**Reply by weslambert** [reply]
It looks like you've attempted to add some `zeek-otx` configuration, and it appears it doesn't like some of the files contained within it's subdirectories. Was this working before, or have you made additional changes?
